.github/workflows/deploy-build-action.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy Artifacts
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: false
+        type: string
+        description: Version to author
+    secrets:
+      SSHKEY:
+        required: true
+
+
+jobs:
+  upload_artifacts:
+    name: Deploy Artifacts ${{ matrix.arch }}-${{ matrix.platform }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - platform: linux-gnu
+            arch: aarch64
+          - platform: linux-gnu
+            arch: x86_64
+          - platform: win
+            arch: x86
+          - platform: win
+            arch: amd64
+          - platform: macos
+            arch: x86_64
+    env:
+      SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+      SSHKEY: ${{ secrets.SSHKEY }}
```

Comment:
We could potentially rely on something like https://github.com/marketplace/actions/webfactory-ssh-agent or https://github.com/marketplace/actions/setup-ssh-agent

Reply:
This is simple enough we don't need a separate action for it.

Reply:
:+1: